central/processlisteningonport/datastore/datastore_impl.go

```diff
@@ -537,7 +538,7 @@ func addNewPLOP(plopObjects []*storage.ProcessListeningOnPortStorage,
 }
 
 func (ds *datastoreImpl) RemovePlopsByPod(ctx context.Context, id string) error {
-	if ok, err := plopSAC.WriteAllowed(ctx); err != nil {
+	if ok, err := adminSAC.WriteAllowed(ctx); err != nil {
```

Comment:
This and the two above should only be used internally. They should never be called as a result of an API call.